lib/pay/billable/stripe.rb

```diff
@@ -20,7 +20,9 @@ def update_stripe_card(token)
         token = ::Stripe::Token.retrieve(token)
 
         return if token.card.id == customer.default_source
-        save_stripe_card(token, customer)
+        result = save_stripe_card(token, customer)
+        self.card_token = nil
```

Comment:
We want to reset the `card_token` attribute here after the token was updated on Stripe.

Reply:
Did this work out for you?? I tried to run this but again got an error. 
![image](https://user-images.githubusercontent.com/20502556/32446590-b2bdefb2-c309-11e7-9ca3-1a3c00e3a1aa.png)

I tried to dive in to the code, but I'm getting pretty confused in the `save_stripe_card(token, customer)` and `update_stripe_card(token)` methods

Reply:
I'm not sure what you mean?